model/project_ref.go

```diff
@@ -996,23 +995,17 @@ func getCommonProjectVariables(projectIds []string) (*ProjectVars, error) {
 			if vars.PrivateVars != nil {
 				commonPrivate = vars.PrivateVars
 			}
-			if vars.RestrictedVars != nil {
-				commonRestricted = vars.RestrictedVars
-			}
 			if vars.AdminOnlyVars != nil {
 				commonAdminOnly = vars.AdminOnlyVars
 			}
 			continue
 		}
 		for key, val := range commonProjectVariables {
-			// if the key is private/restricted in any of the projects, make it private/restricted in the repo
+			// if the key is private/admin only in any of the projects, make it private/admin only in the repo
```

Comment:
I know you didn't write this (and I probably did womp) but could you turn this comment into a full sentence?

Reply:
done!